src/api/firebase.js

```diff
@@ -77,12 +80,20 @@ export async function addItem(listId, { itemName, daysUntilNextPurchase }) {
 	});
 }
 
-export async function updateItem() {
-	/**
-	 * TODO: Fill this out so that it uses the correct Firestore function
-	 * to update an existing item! You'll need to figure out what arguments
-	 * this function must accept!
-	 */
+export async function updateItem(listId, { itemId, isChecked, currentTime }) {
```

Comment:
Something to consider (in general) - how are failures to update handled? This should always be a question when working with external APIs - and network calls specifically)